project-management/src/main/java/life/qbic/projectmanagement/domain/model/project/Contact.java

```diff
@@ -17,9 +17,15 @@ public class Contact {
 
   private String emailAddress;
 
-  public Contact(String fullName, String emailAddress) {
+  private String oidc;
+
+  private String oidcIssuer;
+
+  public Contact(String fullName, String emailAddress, String oidc, String oidcIssuer) {
     requireNonNull(fullName, "fullName must not be null");
     requireNonNull(emailAddress, "emailAddress must not be null");
+    requireNonNull(oidc, "oidc must not be null");
```

Comment:
Why is this required? Can't I have contacts without orcid or other oidc authentication ids?